src/hx711-element.ts

```diff
@@ -0,0 +1,185 @@
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+      >
+        <defs>
+          <path
+            id="e"
+            d="m45 77c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0 25c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0 24c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0 23c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0 24c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0-120c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm244 71c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0 24c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0-49c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm0-24c-3.866 0-7-3.134-7-7s3.134-7 7-7 7 3.134 7 7-3.134 7-7 7zm-0.5 115c-6.3513 0-11.5-5.1487-11.5-11.5s5.1487-11.5 11.5-11.5 11.5 5.1487 11.5 11.5-5.1487 11.5-11.5 11.5zm0-156c-6.3513 0-11.5-5.1487-11.5-11.5s5.1487-11.5 11.5-11.5 11.5 5.1487 11.5 11.5-5.1487 11.5-11.5 11.5zm-288.5-34h340v200h-340v-200z"
+          />
+          <path id="c" d="m0 0h340v200h-340v-200z" />
+          <mask id="f" x="0" y="0" width="340" height="200" fill="white">
+            <use xlink:href="#c" />
+          </mask>
+          <linearGradient id="a" y1="50%" y2="50%">
+            <stop stop-color="#767676" offset="0" />
+            <stop stop-color="#FFFEDF" offset="1" />
+          </linearGradient>
+          <radialGradient id="d" r="37.794%">
+            <stop stop-color="#fff" offset="0" />
+            <stop offset="1" />
+          </radialGradient>
+        </defs>
+        <g fill="none" fill-rule="evenodd">
+          <mask id="b" fill="white">
+            <use xlink:href="#e" />
+          </mask>
+          <g
+            fill="#1C8944"
+            mask="url(#b)"
+            stroke="#ADA216"
+            stroke-dasharray="1,1"
+            stroke-opacity=".53643"
+            stroke-width="4"
+          >
+            <use mask="url(#f)" xlink:href="#c" />
+          </g>
+          <g fill="#D2CDC3" mask="url(#b)">
+            <g transform="translate(36 37)">
+              <rect x="244" y="22" width="17" height="18" />
+              <rect width="17" height="18" />
+            </g>
+          </g>
+          <g mask="url(#b)">
+            <g transform="translate(7 30)">
+              <rect x="25" y="1" width="26" height="147" stroke="#fff" stroke-width="2" />
+              <rect x="270" y="22" width="25" height="102" stroke="#fff" stroke-width="2" />
+              <text
+                font-family="Arial-BoldItalicMT, Arial"
+                font-size="16"
+                font-style="italic"
+                font-weight="bold"
+              >
+                <tspan x="0.05859375" y="69" fill="#FBFBFB">A-</tspan>
+                <tspan x="0.4921875" y="18" fill="#FBFBFB">E+</tspan>
+                <tspan x="297.444824" y="42" fill="#FBFBFB">GND</tspan>
+                <tspan x="297.168945" y="66" fill="#FBFBFB">DT</tspan>
```

Comment:
Shouldn't it be DOUT?

Reply:
I copied the board from this one I have and use. it has `DT` but glad to make the `DOUT`

![IMG_0289 2](https://user-images.githubusercontent.com/1359331/148692864-67991486-9ecb-49c8-8ecc-7f565bc5695b.JPG)

Reply:
Looking at google images, it's either DT or DAT. So we can keep DT - but let's make sure the name in pinInfo matches